application/frontend/tests/auth.setup.ts

```diff
@@ -1,13 +1,43 @@
-import { test as setup } from "@playwright/test"
+import { test as setup, expect } from "@playwright/test"
 import { firstSuperuser, firstSuperuserPassword } from "./config.ts"
 
 const authFile = "playwright/.auth/user.json"
 
 setup("authenticate", async ({ page }) => {
   await page.goto("/login")
+  await page.waitForLoadState("networkidle")
+
+  // Disable animations
+  await page.addStyleTag({
```

Comment:
This can be removed. See below.

Reply:
Commented out to make sure test still passed. Confirmed, and will remove next sprint